app/_src/deck/reference/entities.md

```diff
@@ -10,7 +10,7 @@ It does not manage {{site.base_gateway}} configuration parameters in `kong.conf`
 
 Entity | Managed by decK?
```

Comment:
Not really related to this PR, but would it help to break this table up more into "decK dump" or "decK delete" columns? Since some of them can be managed in one way, but not another?

Reply:
There is no `deck delete` or anything like that. `deck dump`  generates a config file for you, you edit the file however you want, then `deck sync` to your gateway instance to apply configuration. 

There are specific requirements for `workspaces`, and for `document_objects`, which are subsets of the `services` endpoint. With workspaces, it's because a workspace is basically a container for all of your other config; with `document_objects`, it's because they're dev portal entities that happen to be attached to services. 

Does that make sense, and do you think this table still needs some kind of clarification?

Reply:
That makes sense, I don't think the table needs any additional clarification. I was just trying to see if there was a way to break it into additional columns so we wouldn't need footnotes. But footnotes makes the most sense here I think.